spec/cpp_interface.dd

```diff
@@ -770,12 +770,12 @@ $(H2 $(LNAME2 data-type-compatibility, Data Type Compatibility))
 
     $(TROW
     $(ARGS $(B long)),
-    $(ARGS $(B long long))
+    $(ARGS $(B long) if it is 64 bit wide, otherwise $(B long long))
```

Comment:
I would suggest a slightly different wording:
```suggestion
    $(ARGS $(B long) on 64 bits platforms, $(B long long) otherwise)
```

Just wanted to make it more compact, also "if it is 64 bit wide" can be misinterpreted as "long is 64 bits wide", which it always is.

Reply:
That would be true on POSIX, but not on Windows. according to `core.stdc.config`, Windows D `long` == Windows C++ `long`, in both 32-bit and 64-bit.